mnist/mnist.py

```diff
@@ -0,0 +1,287 @@
+# Copyright 2018 Google LLC
```

Comment:
You have 8 GPUs before and after optimizations, why? You have 16 GPUs only after optimizations, why? How are you choosing what gets added?

Reply:
added description to readme